### DISCUSSION TITLE
Improving Import Options for Keyboard Shortcuts in Zed

### DISCUSSION THREAD

**Discussion by sedghi** [zed-industries/zed#21221] [Feature Requests]
Just some thoughts: After a couple of months, I decided to try Zed again, but I hit a wall because none of my custom keyboard shortcuts worked, and I couldn't find any documentation on how to import them. These are general shortcuts, like Delete Line, etc., that I'm pretty sure you can cross-check.

I'm sure there are more users like me. To attract more users, you could provide import options for settings and keyboard shortcuts.

I will check back in another six months.

Cheers

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
